manifests/0000_26_capi-operator_12_clusteroperator.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: cluster-api
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
+  relatedObjects:
+    - group: config.openshift.io
+      name: cluster-api
+      resource: clusteroperators
+    - group: ""
+      name: openshift-cluster-api
+      resource: namespaces
```

Comment:
There are a lot more objects we can be listing in here, eg the deployment, we should make sure that this list is complete